src/test/java/org/pesmypetcare/webservice/controller/petmanager/PetMealRestControllerTest.java

```diff
@@ -0,0 +1,133 @@
+package org.pesmypetcare.webservice.controller.petmanager;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.pesmypetcare.webservice.entity.MealEntity;
+import org.pesmypetcare.webservice.service.MealService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.BDDMockito.willDoNothing;
+import static org.mockito.BDDMockito.willReturn;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@ExtendWith(MockitoExtension.class)
+public class PetMealRestControllerTest {
+    private static String jsonMealEntity;
+    private static String jsonField;
+    private static MealEntity mealEntity;
+    private static List<Map<String, Object>> mealEntityList;
+    private static String owner;
+    private static String petName;
+    private static String date;
+    private static String date2;
+    private static String field;
+    private static String value;
+    private static String urlBase;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private MealService service;
+
+    @BeforeAll
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method length is 18 lines (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5301962)